### ISSUE TITLE
v0. 22.0beta0-foss AndroidTV 

### ISSUE THREAD

**Issue by MarkyMarkDE** [d4rken-org/sdmaid-se#1072]
Since this Version on AndroidTV the main windows looks like horrible, have a look:

![PXL_20240319_005056148](https://github.com/d4rken-org/sdmaid-se/assets/12845477/3aa3011c-bb56-4d61-bb1c-deddd35cf153)

BUG: not able to activate "Bedienungshilfen",  you can do what you like, it stays deactivated.
I'm switched back to 0.21.0 now.

**Comment by MarkyMarkDE** [comment]
oh and only for information:

On Android 14 with the current Patch Update and installation of SDMaid via Browser works again (in December the last time) and it now keep the access for "Bedienungshilfen",  this is absolutely new here.

**Comment by d4rken** [comment]
> As default 2 columns?

Not final but for larger landscape style devices, I like it more than the single column with very stretched items.

Do you like the old view better? What would your ideal design be like for Android TV (or large tablets in landscape?)

> The two columns did not fit in screen, it will little stretched, so not all is visible.

The outside borders didn't change with the update. I made some padding changes, but I think this is not related. Because the settings button and the "pink ribbon" is also cut off, it looks a bit like Android TV is not correctly scaled to your TV size :thinking: Does it line up correctly in other apps?

> If you like access the settings icon (bottom right corner), you do not see, that you are over it, you must click it blind and hope you matched it).
> Planer settings looks different (hover texts on the buttons in dark mode not readable as the buttons are white and the texts also white) and you need set it up different as in 0.21.0.

I'll reproduce this on my TV device and emulator. Thanks :+1: Is this an issue in all themes and modes or only in the dark default theme?

> BUG: not able to activate "Bedienungshilfen", you can do what you like, it stays deactivated. I'm switched back to 0.21.0 now.

After the next update (newer than 0.22.0), if it's reproducible, please record a debug log and a new ticket.

> On Android 14 with the current Patch - Update and installation of SDMaid via Browser works again (in December the last time) and it now keep the access for "Bedienungshilfen", this is absolutely new here (out of Playstore restrictions).

I didn't make any extra changes that would influence this, as far as I know :shrug:

**Comment by MarkyMarkDE** [comment]
>> Do you like the old view better? What would your ideal design be like for Android TV (or large tablets in landscape?)

Yes on AndroidTV you really should let the single column, but okay, maybe I just have to get used to it :wink:

>> The outside borders didn't change with the update. I made some padding changes, but I think this is not related. Because the settings button and the "pink ribbon" is also cut off, it looks a bit like Android TV is not correctly scaled to your TV size 🤔 Does it line up correctly in other apps?

Yes on AndroidTV optimized Apps (not really much, are correct lined up.

>> I'll reproduce this on my TV device and emulator. Thanks 👍 Is this an issue in all themes and modes or only in the dark default theme?

No Problem, for this I'm a paying Beta-tester :satisfied: .
Not sure, I use only the dark mode in both devices :wink:
but the funny, in the old single column version all is okay in dark mode, or I didn't see this here.

>> After the next update (newer than 0.22.0), if it's reproducible, please record a debug log and a new ticket.
okay, I will do that, or you can reproduce it in the meantime for self. 

>> I didn't make any extra changes that would influence this, as far as I know 🤷

Yes, sometimes it is funny how some OS changes can fix something :wink:
I think, this was a BUG on January and February Patch or the last "Feature Drop" on Pixels solves this BUG, don't know :see_no_evil: .
Maybe you can check my log which I send in the past to your @.

**Comment by MarkyMarkDE** [comment]
Yes, it is now little better, than in 0.21.0 but not perfect at all. The main window is too large in 0.23.0 on a HD TV and some buttonstexts in dark mode are not visible when you are "hover" it. But they not white on white buttons anymore that's fine.

**Comment by d4rken** [comment]
Is this still an issue Mark? 

Closing for now, happy to re-open though

**Comment by MarkyMarkDE** [comment]
> Is this still an issue Mark?
> 
> Closing for now, happy to re-open though

okay, no problem :wink:
